### PR TITLE
Login component and Homepage component created. Test written and pass

### PR DESCRIPTION
**Pull request by Seyi-Toluhi**
Login component and Homepage component created. Test written and pass

### CODE REVIEW

File: Frontend/tests/pages/Loginpage.test.jsx
Comment (yulingyou) [inline Frontend/tests/pages/Loginpage.test.jsx:1]: Question: Is commented-out code still waiting to be used, or should it be removed?

Comment (Seyi-Toluhi) [reply Frontend/tests/pages/Loginpage.test.jsx:1]: yes, we still need to use those once the backend is set up. thats why we've left them commented-out.

Comment (yulingyou) [reply Frontend/tests/pages/Loginpage.test.jsx:1]: Sounds good!

**Review by yulingyou — DISMISSED** [review]

File: Frontend/src/pages/Login/loginPage.jsx [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+setEmail(event.target.value);
+};
+
+const handlePasswordChange = (event) => {
+setPassword(event.target.value);
+};
+
+return (
+<>
+    <h2 role="login-heading">Login</h2>
```
Comment (yulingyou) [inline Frontend/src/pages/Login/loginPage.jsx:41]: For better user experience and search engine optimisation, it is suggested to have a title tag on the login page.

Comment (Seyi-Toluhi) [reply Frontend/src/pages/Login/loginPage.jsx:41]: Done but title has been added to Homepage as this is where the login component is rendered

**Pull request closed without merging**